README.de.md

```diff
@@ -0,0 +1,95 @@
+[Patreonseite](https://patreon.com/sircmpwn), oder den 
+[Entwicklungsprämien](https://github.com/SirCmpwn/sway/issues/986) beitragen.
+Jeder ist willkommen, eine Prämie in Anspruch zu nehmen oder für Gewünschtes bereit zu stellen.
+Durch Patreon wird Sways Instandhaltung unterstützt.
+
+## Versionssignaturen
+
+Neue Versionen werden mit 
+[B22DA89A](http://pgp.mit.edu/pks/lookup?op=vindex&search=0x52CB6609B22DA89A) 
+signiert und [auf Github](https://github.com/SirCmpwn/sway/releases) veröffentlicht.
+
+## Status
+
+- [i3-Funktionalitäten](https://github.com/SirCmpwn/sway/issues/2)
+- [IPC-Funktionalitäten](https://github.com/SirCmpwn/sway/issues/98)
+- [i3bar-Funktionalitäten](https://github.com/SirCmpwn/sway/issues/343)
+- [i3-gaps-Funktionalitäten](https://github.com/SirCmpwn/sway/issues/307)
+- [Sicherheitsfunktionalitäten](https://github.com/SirCmpwn/sway/issues/984)
+
+## Installation
+
+### Als Paket
+
+Sway ist in vielen Distributionen verfügbar: versuche das "sway"-Paket deiner zu installieren.
+Falls es nicht vorhanden ist, schau dir 
+[diese Wikiseite](https://github.com/SirCmpwn/sway/wiki/Unsupported-packages) für 
+distributionsspezifische Installationsinformationen an.
+
+Wenn du Interesse hast, Sway für deine Distribution als Paket bereit zu stellen, 
+komm in den IRC-Channel oder schreib eine e-Mail an sir@cmpwn.com (nur englischsprachig).
+
+### Kompilation des Quellcodes
+
+Abhängigkeiten installieren:
+
+* cmake
+* [wlc](https://github.com/Cloudef/wlc)
+* wayland
+* xwayland
+* libinput >= 1.6.0
+* libcap
+* asciidoc
+* pcre
+* json-c
+* pango
+* cairo
+* gdk-pixbuf2 *
+* pam **
+* imagemagick (erforderlich für Bildaufnahme mit swaygrab)
+* ffmpeg (erforderlich für Videoaufnahme swaygrab)
+
+_\*Nur erforderlich für swaybar, swaybg, und swaylock_
+
+_\*\*Nur erforderlich für swaylock_
+
+Führe diese Befehle aus:
+
+    mkdir build
+    cd build
+    cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_SYSCONFDIR=/etc ..
+    make
+    sudo make install
+
+In Systemen mit logind musst du die folgenden Caps der Binärdatei setzen:
+
+    sudo setcap cap_sys_ptrace=eip /usr/local/bin/sway
+    sudo setcap cap_sys_tty_config=eip /usr/local/bin/sway
+
+In Systemen ohne logind musst du die Genehmigungen der Binärdatei setzen:
+
+    sudo chmod a+s /usr/local/bin/sway
+
+## Konfiguration
+
+Falls du i3 benutzt, kopiere deine i3 Konfiguration, andernfalls die Beispielkonfiguration, 
+nach `~/.config/sway/config`. Die Beispielkonfiguration sollte `/etc/sway/config` sein.
+Führe `man 5 sway` für Konfigurationsinformationen aus.
+
+Als Inspiration sind meine eigenen Punktdateien [hier](https://git.sr.ht/~sircmpwn/dotfiles) 
+verfügbar, schau dir auch unbedingt das [Wiki](https://github.com/SirCmpwn/sway/wiki) an.
+
+## Verwendung
+
+Führe `sway` von einer TTY aus. Manche Anzeige-manager könnten funktionieren, werden aber
+nicht von Sway unterstützt (gdm scheint relativ gut zu funktionieren).
```

Comment:
See the change to the english readme in https://github.com/SirCmpwn/sway/pull/1317